### ISSUE TITLE
Map download: Covered region for city maps not visible

### ISSUE THREAD

**Issue by scaidermern** [osmandapp/OsmAnd#15644]
When selecting a region for download on the map OsmAnd will highlight this region in yellow. However, for city maps this doesn't work. Even for large cities no colored area is visible. For the user it is unclear which region will be covered by the map.

Left: city map, right: non-city map.
![photo_2022-11-04_13-50-27](https://user-images.githubusercontent.com/1040784/199977401-80dc1233-1618-4cfb-afc0-7e5cd567b923.jpg)

This seems to happen for all cities, not just Paris.

OsmAnd 4.2.7.

**Comment by vshcherb** [comment]
Yes, we don't select boundaries of city / counties / etc except downloading regions.

**Comment by vshcherb** [comment]
Especially as now we have downloaded regions manually edited here it will be problematic to support all cities, so this feature in a nutshell completely different.
